src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -46,18 +45,29 @@ public static boolean areSameHtml(String expected, String actual, boolean isPart
     
     private static boolean assertSameHtml(String expected, String actual, boolean isPart,
                                           boolean isDifferenceToBeShown) {
-        String processedExpected = convertToStandardHtml(expected, isPart);
+        String processedExpected = expected.replace("\r", "");
```

Comment:
I just realized I can use `.replace(Const.EOL, "\n")` which should apply to all systems.